packages/wallet/src/App.js

```diff
@@ -25,13 +25,13 @@ const App = () => {
       <Suspense fallback={<NotFound />}>
         <Routes>
           <Route path='/' element={<Wallet />} />
-          <Route path='/assets' element={<Navigate to='/assets/overview' replace />} />
-          <Route path='/assets/overview' element={<Assets />} />
+          <Route path='/home' element={<Navigate to='/home/overview' replace />} />
+          <Route path='/home/overview' element={<Home />} />
```

Comment:
is there any problem with just having the one `/home` route and deleting `/home/overview`?

Reply:
I think we'll want more pages under `home` so I feel comfortable leaving this here.